src/qml/editorwidgets/DateTime.qml

```diff
@@ -37,20 +37,23 @@ Item {
       Layout.minimumHeight: 48 * dp
 
       Rectangle {
-        anchors.fill: parent
         id: backgroundRect
+        anchors.fill: parent
         border.color: label.activeFocus ? "#17a81a" : "#21be2b"
         border.width: label.activeFocus ? 2 : 1
-        color: "#dddddd"
+        color: "transparent"
         radius: 2
+        visible: enabled
       }
 
       TextField {
         id: label
 
-        anchors.fill: parent
+        anchors.left: parent.left
+        anchors.right: parent.right
         verticalAlignment: Text.AlignVCenter
-        font.pointSize: 16
+        font.pointSize: 18 * dp
```

Comment:
Are you sure the **point**Size needs to be multiplied with dp? I think it's only the **pixel**Size and the pointSize is fine, just like that.